templates/inbox.html

```diff
@@ -40,9 +40,9 @@
     <ng-include src="'templates/modals/tour_select.html'" onload="setupTour()"></ng-include>
     <ng-include src="'templates/modals/update_facility.html'"></ng-include>
     <ng-include src="'templates/modals/update_password.html'"></ng-include>
-    <ng-include src="'templates/modals/user_language.html'" onload="setupUserLanguage()"></ng-include>
     <ng-include src="'templates/modals/version_update.html'"></ng-include>
     <ng-include src="'templates/modals/welcome.html'" onload="setupWelcome()"></ng-include>
+    {{setupUserLanguage()}}
```

Comment:
This doesn't make sense to me. The ng-include onload parameter defers execution until the template has finished rendering, but calling it here instead just invokes it immediately. If it can now be executed immediately we should just invoke it in the controller, rather than call it from the template.